### PR TITLE
Added missing dendro reactions

### PR DESCRIPTION
**Pull request by majkus522**

### CODE REVIEW

File: assets/data/elements/electro/en.json [hunk 1]
```diff
@@ -16,6 +16,11 @@
       "name": "Superconduct",
       "elements": ["Cryo"],
       "description": "When Electro meets Cryo, Superconduct occurs. Superconduct deals AoE Cryo DMG and significantly decreases Physical RES for the affected being."
+    },
+    {
+      "name": "Catalyze",
+      "elements": ["Electro"],
```
Comment (man90es) [inline assets/data/elements/electro/en.json]: The array should have "Dendro", not "Electro"

File: assets/data/elements/hydro/en.json [hunk 2]
```diff
@@ -16,6 +16,11 @@
       "name": "Frozen",
       "elements": ["Cryo"],
       "description": "When Hydro meets Cryo, Frozen occurs. Frozen beings are rendered immobile, but also become rock-hard."
+    },
+    {
+      "name": "Bloom",
+      "elements": ["Hydro"],
```
Comment (man90es) [inline assets/data/elements/hydro/en.json]: The array should have "Dendro", not "Hydro"

**Review by man90es — CHANGES_REQUESTED** [review]

**Comment by majkus522** [comment]
Fixed.
Also how should i add reactions like hyperbloom

**Comment by AkiVonAkira** [comment]
> Fixed. Also how should i add reactions like hyperbloom

Maybe something like (?);
      "name": "Hyperbloom",
      "elements": ["Hydro", "Electro"],
      "description": ....

**Comment by majkus522** [comment]
But how are you gona differentiate between reactions that can be triggered with multiple elements (e.g. Swirl can be triggered with cryo or pyro or hydro or electro). In hyperbloom you need to first trigger bloom then add electro.

**Review by man90es — APPROVED** [review]

**Pull request merged**